src/main/webapp/app/exercises/programming/manage/grading/feedback-analysis/Modal/feedback-modal.component.scss

```diff
@@ -0,0 +1,23 @@
+.modal-body {
+    padding: 1rem;
+}
+
+.modal-body p {
+    margin-bottom: 0;
+}
+
+.modal-label {
+    font-weight: bold;
+}
+
+.border {
```

Comment:
Checkout the bootstrap border classes https://getbootstrap.com/docs/5.0/utilities/borders/